pkgs/regen.nix

```diff
@@ -0,0 +1,53 @@
+let
+  localLib = import ../lib.nix;
+  jemallocOverlay = self: super: {
+    # jemalloc has a bug that caused cardano-sl-db to fail to link (via
+    # rocksdb, which can use jemalloc).
+    # https://github.com/jemalloc/jemalloc/issues/937
+    # Using jemalloc 510 with the --disable-initial-exec-tls flag seems to
+    # fix it.
+    jemalloc = self.callPackage ../nix/jemalloc/jemalloc510.nix {};
+  };
+in
+{ system ? builtins.currentSystem
+, config ? {}
+, pkgs ? (import (localLib.fetchNixPkgs) { inherit system config; overlays = [ jemallocOverlay ]; })
```

Comment:
This repetition is going to get cleaned up in another PR.

Reply:
Can we just clean it up here?
```
, iohkPkgs ? import ../ { inherit config system gitrev; }
, pkgs ? iohkPkgs.pkgs
```

Then `jemallocOverlay` can be dropped as well.

Reply:
Ah I was looking for `localLib.pkgs` and couldn't find it. Will fix now.